test/unit/argument_parser/detail/version_check_test.hpp

```diff
@@ -94,6 +91,12 @@ struct version_check : public ::testing::Test
         return seqan3::detail::version_checker{app_name, std::string{}}.timestamp_filename;
     }
 
+    std::chrono::duration<long int>::rep current_unix_timestamp()
+    {
+        return std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch())
```

Comment:
I think a `using std::chrono` locally here would probably improve the readability a lot. But only if you like :nail_care:

Reply:
I wouldn't